client/components/singleProduct.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import {fetchProduct} from '../store/singleProduct'
+import {Link} from 'react-router-dom'
+
+/**
+ * COMPONENT
+ */
+
+class SingleProduct extends React.Component {
+  componentDidMount() {
+    this.props.getProduct(this.props.match.params.id)
+  }
+
+  render() {
+    const {product} = this.props
+
+    return (
+      <div>
+        <img src={product.imageUrl} style={{width: '300px'}} />
+        <h2>{product.name}</h2>
+        <h3>Price: ${product.price / 100}</h3>
+        <p>{product.description}</p>
+        {/* choose on of the following, or delete both if we run out of time */}
+        <input
```

Comment:
I like this one better 👍  but maybe instead of a placeholder that says Qty we can just start at 1

Reply:
I meant the input tag :)